inst/shiny/modules/units_table.R

```diff
@@ -0,0 +1,205 @@
+units_table_ui <- function(id) {
+  ns <- NS(id)
+  # Button to open a module message with the parameter units table #
+  tagList(
+    actionButton(
+      ns("open_units_table"),
+      icon = shiny::icon("scale-balanced"),
+      label = "Parameter Units",
+      disabled = TRUE
+    )
+  )
+}
+
+units_table_server <- function(id, mydata, res_nca = reactive(NULL), params_to_calculate) {
```

Comment:
issue:
```suggestion
units_table_server <- function(id, mydata, res_nca = reactiveVal(NULL), params_to_calculate) {
```
as the module expects `reactiveVal` as opposed to `reactive`